recipes/apache-solr/LICENSE.txt

```diff
@@ -0,0 +1,226 @@
+                                 Apache License
```

Comment:
If this _really_ isn't in the distribution (unlikely for a flagship Apache project), I recommend getting it (and the NOTICE) from:

```yaml
- url: https://github.com/apache/solr/blob/releases/solr/{{ version }}/LICENSE.txt
  sha256: ... # only have to get it once, the bot will update it in the future
- url: https://github.com/apache/solr/blob/releases/solr/{{ version }}/NOTICE.txt
  sha256: ... # only have to get it once, the bot will update it in the future
```

Reply:
Ok. I've added instructions to the recipe to download those files from the solr git repository.